src/device-state.coffee

```diff
@@ -41,6 +44,9 @@ validateState = Promise.method (state) ->
 	if state.dependent?
 		validateDependentState(state.dependent)
 
+rimraf = (p) ->
+	execAsync("rm -rf \"#{p.replace(/"/g, '\\"')}\"")
```

Comment:
Is there a possibility that the backup file can have quotes?

What worries me about this is that it won't cover all escaping necessary for shell commands, but it looks "general". I think we should either be very target (e.g. know exactly that a path will not contain bad chars) or support everything (maybe switch back to rimraf if we cannot guarantee the former? (sorry!))

Reply:
I don't know if there's chances for quotes, but I wouldn't like creating a function that can't handle such a simple thing... and it's the kind of thing that could fail in the future if things change. So yeah, let's go back to rimraf :)